### DISCUSSION TITLE
[Ignore the title] This browser works great with 8K vaapi decoding, Firefox doesn't anymore

### DISCUSSION THREAD

**Discussion by SheMelody** [Alex313031/Mercury#197] [Show and tell]
~~I just want to say that, basically, I've found out that regular Firefox has a regression in the latest update which makes it unable to hardware decode 8K videos (AV1) for more than 3-5 seconds without crashing and stopping the video.~~

~~**On the other hand, Mercury seems to not be affected by this regression.**~~

~~Since nobody has mentioned this, I just wanted to mention this so that this won't happen with the next versions.~~

Nevermind, this fork is also affected, we will have to wait for a fix. It just took more time to trigger itself.

**Comment by gz83** [comment]
Hardware acceleration on Linux is pretty bad and usually requires a lot of tweaking to even have a chance of enabling hardware acceleration.

**Reply by SheMelody** [reply]
Nevermind, this fork is also affected by that regression, it just took a few minutes to trigger. Firefox devs have to fix this on their own.

Edit: Anyway it's not as catastrophic as you described it, it's fairly easy to use hardware decoding in Firefox based browsers. Documentations such as the Arch Wiki help a lot in setting everything up. It's still pretty great with 4K 60 fps.
